packages/cloudfare-kv/src/index.js

```diff
@@ -0,0 +1,120 @@
+'use strict'
+
+const fetch = import('node-fetch').then(mod => mod.default)
```

Comment:
can we use [`isomorphic-unfetch`](https://github.com/developit/unfetch/tree/master/packages/isomorphic-unfetch) instead?

It will be possible to use the library under an [Edge Function](https://vercel.com/edge) scenario, just using node-fetch when is necessary.

Reply:
sounds good